Dfe.Data.SearchPrototype/Web/Controllers/HomeController.cs

```diff
@@ -60,6 +60,9 @@ public async Task<IActionResult> Index(string searchKeyWord)
             await _searchByKeywordUseCase.HandleRequest(
                 new SearchByKeywordRequest(searchKeyWord + "*", "establishments"));
 
+        // Currently returns just the establishment data/results.
+        // Once this also starts returning the facets too, we will need to
+        // update the mapper to map the facets also.
```

Comment:
Looks like this might have been done in the mapper now, so this comment can likely be removed?